dmscripts/mark_definite_framework_results.py

```diff
@@ -0,0 +1,135 @@
+from collections import Counter
+import logging
+
+
+import jsonschema
+
+
+def _passes_validation(candidate, schema, logger, schema_name="schema", tablevel=0, loglevel=logging.INFO):
```

Comment:
Yup, I followed the existing precedent, the tablevel is because the same function is called both under the supplier_framework and draft service contexts, which work at different tablevels.